### PR TITLE
Update locales-mn-MN.xml

### PR DESCRIPTION
**Pull request by munkhmagnai**

### CODE REVIEW

**Comment by adam3smith** [comment]
Somehow we missed a couple of locale pull requests. Sorry about that. This looks great, thanks!

File: locales-mn-MN.xml [hunk 1]
```diff
@@ -70,37 +70,37 @@
...
-    <term name="ordinal-03">rd</term>
-    <term name="ordinal-11">th</term>
-    <term name="ordinal-12">th</term>
-    <term name="ordinal-13">th</term>
+    <term name="ordinal-01">1-р</term>
+    <term name="ordinal-02">2-р</term>
+    <term name="ordinal-03">3-р</term>
+    <term name="ordinal-11">11-р</term>
+    <term name="ordinal-12">12-р</term>
+    <term name="ordinal-13">13-р</term>
```
Comment (rmzelle) [inline locales-mn-MN.xml]: @adam3smith, my Mongolian is a bit rusty, but this looks a bit suspicious. Should probably just be `<term name="ordinal">-p</term>` as sole ordinal term. Agreed?

Comment (rmzelle) [reply locales-mn-MN.xml]: @adam3smith, bump.

Comment (adam3smith) [reply locales-mn-MN.xml]: oh yes, sorry. 
image

Comment (rmzelle) [reply locales-mn-MN.xml]: I'll prepare a fix.

Comment (rmzelle) [reply locales-mn-MN.xml]: (fixed in #189)

**Pull request merged**